lldb/source/Plugins/Language/Swift/SwiftLanguage.cpp

```diff
@@ -889,8 +889,11 @@ std::vector<ConstString> SwiftLanguage::GetPossibleFormattersMatches(
 
   const bool check_cpp = false;
   const bool check_objc = false;
-  bool canBeSwiftDynamic = compiler_type.IsPossibleDynamicType(
-      nullptr, check_cpp, check_objc);
+  bool canBeSwiftDynamic =
+      compiler_type.IsPossibleDynamicType(nullptr, check_cpp, check_objc) ||
+      // Some CoreFoundation types may be Swift types disguised as ObjC objects.
+      (compiler_type.GetCanonicalType().GetTypeClass() ==
+       eTypeClassObjCObjectPointer);
```

Comment:
this `eTypeClassObjCObjectPointer` check is twice in the PR, should it be made less adhoc somehow? Should this check be moved into  `IsPossibleDynamicType`? Or are there cases where `eTypeClassObjCObjectPointer` is not dynamic?